app/views/shared/review_application/_income_payments_and_assets.html.erb

```diff
@@ -27,7 +27,7 @@
     <section>
       <%= govuk_summary_list(actions: false, classes: "govuk-!-margin-top-6") do |summary_list| %>
         <%= summary_list.with_row(html_attributes: { id: "app-check-your-answers__student_finance_question" }) do |row| %>
-          <%= row.with_key(text: t("shared.check_answers.student_finance.does_your_client"), classes: "govuk-!-width-one-half") %>
+          <%= row.with_key(text: t("shared.check_answers.student_finance.does_your_client", individual_text: "your client"), classes: "govuk-!-width-one-half") %>
```

Comment:
Should the student finance block be repeated for Partner?

Reply:
This is on the review application page, which I've not updated as part of this ticket. There's a few places where I've hardcoded the individual as `applicant` as the `partner` bits haven't been added yet. I assume this'll be added later (and if we don't have a ticket for it, we should probably add one)